instrumentation/slowrendering/src/main/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderingDetectorInstrumentation.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.slowrendering;
+
+import android.app.Application;
+import androidx.annotation.NonNull;
+import com.google.auto.service.AutoService;
+import io.opentelemetry.android.OpenTelemetryRum;
+import io.opentelemetry.android.instrumentation.AndroidInstrumentation;
+import java.time.Duration;
+
+/**
+ * A builder of {@link SlowRenderingDetector}.
+ *
+ * <p>This class is internal and not for public use. Its APIs are unstable and can change at any
+ * time.
+ */
+@AutoService(AndroidInstrumentation.class)
+public final class SlowRenderingDetectorInstrumentation implements AndroidInstrumentation {
```

Comment:
Similar to the network, this simply becomes `SlowRenderingInstrumentation` or `SlowRenderInstrumentation`.